config/default_records/request_types/004_limber_high_throughput.yml

```diff
@@ -38,6 +38,7 @@ limber_reisc:
     - LB Lib PCR-XP
     - LTN Lib PCR XP # for Targeted NanoSeq ReISC
     - RVI Lib PCR XP # for RVI Bait Capture Library
+    - RVIG Lib PCR XP # for RVIG Bait Capture Library
```

Comment:
it may be that record loader won't automatically try to update the acceptable purpose added in this file, in which case you could separate the RVIG Lib PCR XP purpose into a wip file.
On the other hand, what's the harm if you define that purpose and have it in here if there's no way to use that plate type.
I think the PR is good as is.

Reply:
You will want to check what it does on deployment, in case you need to manually add that acceptable purpose via server rails console.